keyboards/ploopyco/madromys/madromys.h

```diff
@@ -33,5 +33,5 @@ extern uint16_t          dpi_array[];
 
 enum ploopy_keycodes {
     DPI_CONFIG = QK_KB_0,
-    DRAG_SCROLL,
+    DRAG_SCROLL, TOGGLE_SCROLL
```

Comment:
This will let people define the behavior for scrolling with VIA. They could even have both keys if they want. I wasn't able to get TOGGLE_SCROLL to be accepted in the UI but you can edit the layout.json to have `"CUSTOM(2)"` for a key to make it toggle scroll.